fastestimator/pipeline.py

```diff
@@ -201,9 +201,19 @@ def benchmark(self, mode: str = "train", epoch: int = 1, num_steps: int = 1000,
                         duration_list[i] += duration
 
             total_time = np.sum(duration_list)
+            max_op_len = max(len(op.__class__.__name__) for op in op_list)
+            max_in_len = max([len(str(op.inputs)) - 2 for op in op_list] + [len("Inputs")])
```

Comment:
could save some print characters by doing ", ".join(op.inputs) rather than str(op.inputs)

Reply:
Sure. I will make that change.